src/gallium/drivers/iris/iris_resolve.c

```diff
@@ -215,6 +223,9 @@ iris_predraw_resolve_framebuffer(struct iris_context *ice,
       if (zs_surf) {
          struct iris_resource *z_res, *s_res;
          iris_get_depth_stencil_resources(zs_surf->texture, &z_res, &s_res);
+         if (z_res->base.b.depth_cleared) {
+            z_res->base.b.depth_cleared = false;
```

Comment:
a little confused about this depth_cleared, when it is true, why set it as false here?